server/datastore/mysql/labels_test.go

```diff
@@ -1812,171 +1814,84 @@ func labelIDFromName(t *testing.T, ds fleet.Datastore, name string) uint {
 	return 0
 }
 
-func testUpdateLabelMembershipByHostIDs(t *testing.T, ds *Datastore) {
-	ctx := context.Background()
-	filter := fleet.TeamFilter{User: test.UserAdmin}
+func testUpdateLabelMembershipForTransferredHost(t *testing.T, ds *Datastore) {
```

Comment:
This diff would make a lot more sense if testUpdateLabelMembershipByHostIDs hadn't moved. Threw the following at Zed + Opus 4.5 to clue me in that we're looking at a move + some new tests rather than significant modifications to an existing test (Prompt: In datastore/mysql/labels_test.go, what changed between 36781-team-labels-backend and main? The diff is confusing.) but I'm concerned for git blame confusion later, particularly since this'll get squashed on merge so even if you have a "move test" commit in this branch it won't make it to `main`.